crates/chainspec/src/spec.rs

```diff
@@ -22,6 +21,7 @@ use reth_primitives_traits::{
 use reth_trie_common::root::state_root_ref_unhashed;
 #[cfg(feature = "std")]
 use std::sync::Arc;
+use std::sync::LazyLock;
```

Comment:
This file has std and no std specific elements so I think you need to do something like (should be true for other files with same cfg config)

```rust
#[cfg(feature = "std")]
use std::sync::LazyLock;

#[cfg(not(feature = "std"))]
use alloc::sync::LazyLock;
```

Reply:
I was troubled beacuase I thought I will have to manually implement the no-std of std::sync::LazyLock somehow, so as to not use alloc crate at all! Thank you!

Reply:
LazyLock is only available in std. It does not exist in alloc, as it depends on the OS and other system-specific stuff that are only available in std.

You must keep once_cell in the no_std case.

I would also recommend you run the command locally to see the failures instead of pushing a million commits one at a time and waiting for someone to run CI on it

Reply:
@DaniPopes `Caused by:
  No space left on device (os error 28)` is what's stopping me from running those commands :( Mac M1 256 GB was not the right choice! 
  
 Will try on gitpod!